c2corg_ui/templates/outing/edit.html

```diff
@@ -23,10 +23,11 @@ <h1 ng-cloak ng-init="id=${outing_id if outing_id else 0}" ng-bind="id ? ('Editi
   % endif 
     name="editForm" novalidate ng-submit="editCtrl.submitForm(editForm.$valid)" class="document-editing">
     
-    ## Init max steps. It won't work with ng-init
+    ## It won't work with ng-init
     <span class="ng-hide">
       {{editCtrl.max_steps = 5}} 
       {{document = outing}}
+      {{quality_types = [ 'stub', 'medium', 'correct', 'good', 'excellent']}}
```

Comment:
Hard coding values is evil!
It would be better to retrieve those values from ``c2corg_common/attributes.py`` even though it is not available in this file yet. I have open an issue in the API: https://github.com/c2corg/v6_api/issues/213